src/index.ts

```diff
@@ -20,7 +20,7 @@ function getServiceData(apiName: string, beaconName: string, chain: string) {
 
   const goBeaconData = goSync(() => require(`@api3dao/operations/data/apis/${apiName}/beacons/${beaconName}`));
   if (!isGoSuccess(goBeaconData)) throw new Error(`Service data does not exist.`);
-  const beaconData = JSON.parse(goBeaconData.data).toString();
+  const beaconData = goBeaconData.data;
```

Comment:
This was a bug before - since after the data is imported it is already parsed into JS object - not JSON text.